cmd/instances.go

```diff
@@ -21,6 +21,10 @@ var flagEmail string
 var flagApps []string
 var flagDev bool
 
+func validDomain(domain string) bool {
+	return !strings.ContainsAny(domain, " /?#@\t\r\n")
+}
```

Comment:
why not also put these validations on the [backend side](https://github.com/cozy/cozy-stack/blob/master/pkg/instance/instance.go#L317-L326) ?